src/borg/testsuite/archiver/key_cmds.py

```diff
@@ -12,8 +12,10 @@
 from ...repository import Repository
 from .. import environment_variable
 from .. import key
-from . import RK_ENCRYPTION, KF_ENCRYPTION, cmd, _extract_repository_id, _set_repository_id
-from . import pytest_generate_tests  # NOQA
+from . import RK_ENCRYPTION, KF_ENCRYPTION, cmd, _extract_repository_id, _set_repository_id, generate_archiver_tests
+
+# Tests that include the 'archivers' argument will generate a tests for each kind of archivers specified.
```

Comment:
you could move that to the docstring of generate_archiver_tests, so you don't need to duplicate this comment everywhere.

Reply:
Done!